### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by DmitriyKondaurov**
screencapture-localhost-3000-geekhub-frontend-js_2018-hw3-index-ru-html-2018-11-04-04_40_28

### CODE REVIEW

**Comment by yana-tsylinska** [comment]
підключи Font Awesome лінкою пліз - https://fontawesome.com/how-to-use/on-the-web/setup/getting-started?using=web-fonts-with-css
я не змогла добратись до твого коду ((

**Comment by yana-tsylinska** [comment]
https://cdnjs.com/libraries/font-awesome :wink:

**Comment by spolischook** [comment]
It's impossible to review such big PR. 
Make few small rather then one god PR

**Review by spolischook — CHANGES_REQUESTED** [review]
Move dependencies to package.json

**Comment by DmitriyKondaurov** [comment]
> It's impossible to review such big PR.
> Make few small rather then one god PR

PR - it is project ?

I did not understand,  what mistakes exactly I made? 
Not enough commits? They are too large?

In package.json I have :
      "devDependencies": {
          "bower": "^1.8.4",
          "browser-sync": "^2.26.3",
          "gulp": "^3.9.1"
        },
        "dependencies": {
          "jquery": "^3.3.1",
          "normalize.css": "^8.0.0",
          "owl.carousel": "^2.3.4"
        }

What else I must put in dependencies ?

**Review by spolischook — APPROVED** [review]

File: HW3/index-ru.html [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" integrity="sha256-NuCn4IvuZXdBaFKJOAcsU2Q3ZpwbdFisd5dux4jkQ5w=" crossorigin="anonymous" />
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+<header class="header">
+    <div class="header_wrapper">
+        <h1 class="logo">
+            <a href="#hero">
+                <img class="logo-img" src="img/logo.png" alt="Photo of Taras Shevchenko, famous ukrainian writer">
+                <span class="logo-caption">Театр</span>
```
Comment (yana-tsylinska) [inline HW3/index-ru.html:20]: цей текст має бути частиною лого - тобто частиною картинки, назву сайту ти просто ховаєш за лого і не показуєш))

File: HW3/index-ru.html [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    <div class="hero_carousel">
+        <ul class="hero_carousel_event-img-list">
+            <li><img class="hero_carousel_event-img-item hide" src="img/event/event-1.png" alt="Event image"></li>
+            <li><img class="hero_carousel_event-img-item hide" src="img/event/event-2.png" alt="Event image"></li>
+            <li><img class="hero_carousel_event-img-item" src="img/event/event-3.png" alt="Event image"></li>
+            <li><img class="hero_carousel_event-img-item hide" src="img/event/event-4.png" alt="Event image"></li>
+            <li><img class="hero_carousel_event-img-item hide" src="img/event/event-5.png" alt="Event image"></li>
+        </ul>
+        <nav class="hero_carousel_event-nav">
+            <div class="hero_carousel_event-nav_left" title="Previous">&lsaquo;<span class="hide">Previous</span></div>
```
Comment (yana-tsylinska) [inline HW3/index-ru.html:51]: гарно зроблено, тільки prev/next краще через button робити))

**Pull request merged**